src/containers/downloads/DownloadCasesModal.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+
+import { List, Map } from 'immutable';
+import {
+  ActionModal,
+  Checkbox,
+  Label,
+  Select,
+} from 'lattice-ui-kit';
+import { ReduxUtils } from 'lattice-utils';
+import { RequestStates } from 'redux-reqseq';
+
+import { DOWNLOAD_CASES, downloadCases } from './actions';
+
+import { resetRequestState } from '../../core/redux/actions';
+import { DownloadsReduxConstants, REQUEST_STATE } from '../../core/redux/constants';
+import { getPersonName } from '../../utils/people';
```

Comment:
a good point! actually `REQUEST_STATE` is imported from `lattice-utils` in the `'../../core/redux/constants'` file, so that in each file that needs `REQUEST_STATE`, i can have fewer imports/consts.